pkgs/by-name/ig/igir/package.nix

```diff
@@ -21,6 +26,14 @@ buildNpmPackage rec {
     patchShebangs scripts/update-readme-help.sh
   '';
 
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [ stdenv.cc.cc.lib ];
+
+  # from lib/node_modules/igir/node_modules/@node-rs/crc32-linux-x64-musl/crc32.linux-x64-musl.node
+  # I don't know what "crc32.linux-x64-musl.node" is
```

Comment:
You can drop this line. It's just some native extension of that node module, but for the musl libc, irrelevant for our use here.

Reply:
Ah ok, I've changed it to say that it's irrelevant for us